bin/pytorch_inference/Main.cc

```diff
@@ -290,14 +293,14 @@ int main(int argc, char** argv) {
         return EXIT_FAILURE;
     }
 
-    if (numLibTorchThreads != -1) {
-        at::set_num_threads(numLibTorchThreads);
-    }
-    if (numLibTorchInterOpThreads != -1) {
-        at::set_num_interop_threads(numLibTorchInterOpThreads);
-    }
+    at::set_num_threads(inferenceThreads);
+
+    // This is not used as we don't call at::launch anywhere.
+    // Setting it to 1 to ensure there is no thread pool sitting around.
+    at::set_num_interop_threads(1);
+
     LOG_DEBUG(<< at::get_parallel_info());
-    LOG_DEBUG(<< "Number of parallel forwarding threads: " << numParallelForwardingThreads);
+    LOG_DEBUG(<< "Model threads: " << modelThreads);
```

Comment:
Similarly, maybe we should constrain this to be at most `std::thread::hardware_concurrency`.  (0 is already equivalent to 1 for this one.)